cmd/root.go

```diff
@@ -105,7 +105,11 @@ func NewRootCommand() *cobra.Command {
 	command.PersistentFlags().BoolVar(
 		&params.Force, "force", false,
 		"Don't ask for confirmation before deleting resources. "+
-			"Instead it waits 15s before continuing.")
+			"Instead it waits 15s before continuing. Set --force-sleep to change the wait time.")
+	command.PersistentFlags().IntVar(
```

Comment:
There is a function that outputs a duration directly: https://godoc.org/github.com/spf13/pflag#DurationVar

Reply:
True, but doing this forces the user to know to type "15s" instead of "15". If we foresee users wanting to set durations that aren't easily convertible into seconds (60, 300, etc.), then it makes sense, otherwise I feel it makes more sense to just use seconds.